draft-ietf-httpbis-connect-tcp.md

```diff
@@ -122,6 +122,12 @@ HEADERS
 ~~~
 {: title="Templated TCP proxy example in HTTP/2"}
 
+## Use of Relevant Headers
+
+A templated TCP proxy has an unambiguous origin of its own.  Origin-scoped HTTP header fields such as Alt-Svc {{?RFC7838}} and Strict-Transport-Security {{?RFC6797}} apply to this origin.
+
+In a web browser, authentication for templated TCP proxy requests uses a "407 (Proxy Authentication Required)" response, and the "Proxy-Authenticate" and "Proxy-Authorization" header fields.  Non-proxy-related requests to the same origin continue to use "WWW-Authenticate" and "Authorization" header fields.  This distinction is not technically required, but it may facilitate separation of credentials between the browser and the webpage contents.
```

Comment:
I have a few concerns related to this paragraph. First, this isn't specific to Web browsers. Second, this should be written as normative text, as it isn't clear to me if it is trying to convey MUST or SHOULD. Third, I think both Proxy-Authorization and Authorization make sense here just like they both make sense for GET - because you can have an intermediary between the client and proxy. But overall, this topic warrants its own discussion. Can we split this off to its own issue and PR?

Reply:
Agreed that if we are to add this paragraph we need to open a specific issue.

FWIW, [RFC 9110 Section 11.6](https://www.rfc-editor.org/rfc/rfc9110.html#name-authenticating-users-to-ori) states that WWW-Authenticate, Authorization, and 401 deals with origins (or the target resource). I would personally be surprised if we are to use the proxy variants here.

Reply:
OK, I removed this paragraph and opened #2694 to discuss.